runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -43,50 +50,170 @@ void configureHeader();
 void insertNoOpCommand(unsigned int numPadBytes);
 }
 
-#define OSTREAM_OP(O_TYPE, TYPE) O_TYPE &operator<<(O_TYPE &os, const TYPE &s);
+#define XAIE2_BASE_ADDR 0x40000000
+#define XAIE1_BASE_ADDR 0x20000000000
+#define XAIE1_COL_SHIFT 23
+#define XAIE1_ROW_SHIFT 18
+#define XAIE2_BASE_ADDR 0x40000000
+#define XAIE2_COL_SHIFT 25
+#define XAIE2_ROW_SHIFT 20
+#define XAIE_MEM_TILE_ROW_START 1
+#define XAIE_NUM_MEM_TILE_ROWS 1
+#define XAIE_NUM_SHIM_TILE_ROWS 1
+#define XAIE_PARTITION_BASE_ADDR 0x0
+#define XAIE_SHIM_ROW 0
+
+#define NPI_ADDR 0x0
+#define NUM_LOCKS 16
+#define MEM_TILE_LOCK_ID_INCR 64
+#define BASE_ADDR_A_INCR 0x80000
```

Comment:
Should these be added as a config to `aie-rt` itself? It seems like there is no existing AIE2IPU config inside that repo?

Reply:
[There is](https://github.com/Xilinx/aie-rt/blob/release/main_aig/driver/src/global/xaie2ipugbl_reginit.c) but these things are (for the most part) not part of that config. I have tried to find as many of these constants as possible (or find their representation in terms of more fundamental constants) but I believe the philosophy here is that `aie-rt` is parameterized by these values. Anyway I will go over these constants another time and I see if I can reduce their number even further.

Reply:
Combing with your pointer `xaie_lite_hwcfg.h` I guess it's correct there's no such config in `aie-rt`. I propose we carry it locally for now and upstream/PR it to them.

Reply:
Yeah, makes sense

Reply:
Added `xaie_hwcfg.c/h`.